.github/workflows/ci.yaml

```diff
@@ -19,14 +19,18 @@ jobs:
       matrix:
         config:
           - os: "ubuntu-latest"
-            extension: ""
             platformAgnosticChecks: true
-          - os: "macos-latest"
-            extension: ""
-            extraArgs: ""
+          - {
```

Comment:
stupid question: do we really need curly braces to define a map of key value pairs?